Automattic/jetpack
# VideoPress: removed deprecated wp-block-bridge lib

This PR is the last step to remove the deprecated wp-block-bridge, closing this issue: https://github.com/Automattic/jetpack/issues/29070

Fixes https://github.com/Automattic/jetpack/issues/29070

## Proposed changes:

* VideoPress: removed deprecated wp-block-bridge lib

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check the app works as expected. The wp-block-bridge script is not used so removing it should not change anything.
* You can do a deep search of the `wp-block-bridge` in the source code. Nothing should be found.
